appunite/Loudius
# [LD-165] Change name of "success" param to "action_success".

# Pull-Request
## Description

### Why?

We don't see the success parameter in the looker studio because it is a built-in parameter.

### What?

Change the name of "success" parameter to "action_success".
Add "action_success" parameter to custom definitions.

## Links to related issues

- Fixes [LD-165](https://loudius.atlassian.net/browse/LD-165)


## Demo


| Before the PR | After the PR |
|---------------|--------------|
|               |              |

## How to test

Check out the [looker studio dashboard](https://lookerstudio.google.com/reporting/486e368b-d491-4807-b926-e82f99d22708/page/WDBqD) to see the success rate of actions.

## Documentation
- [Analytics Documentation](https://firebase.google.com/docs/reference/kotlin/com/google/firebase/analytics/FirebaseAnalytics.Param)
- [Firebase Console](https://console.firebase.google.com/project/loudius-fce48/overview)
- [Looker Studio Dashboard](https://lookerstudio.google.com/reporting/486e368b-d491-4807-b926-e82f99d22708/page/WDBqD)

## Checklist


- [x] Functionality is covered by unit tests
- [x] Functionality is covered by integration tests
